### DISCUSSION TITLE
DBeaver MSSQL connection is in readonly mode by default

### DISCUSSION THREAD

**Discussion by Kevinpandya953** [dbeaver/dbeaver#35378] [Q&A] [answered]
### Description

I am trying to create an MSSQL server connection on academic license using mannual connection type with ODBC driver and windows authentication. 

Whenever I open a database object such as tables, it always opens into readonly mode and I am not able to fix that issue by myself. 
Also whenever database object is opened, it always opens as a view and mock data generator also does not work.

![issue 1](https://github.com/user-attachments/assets/da3f1d58-7734-4b36-9cc5-1df8fc9f85d9)
![issue1 1](https://github.com/user-attachments/assets/74bbf693-c484-47e9-8f5d-ae8522950071)
![issue1](https://github.com/user-attachments/assets/b30d2244-60bd-42b1-8bd6-b1757a557341)

### DBeaver Version

DBeaver Enterprise 24.0.0

### Operating System

Windows 11

### Database and driver

Database Name : SQL Server 15.0.2116 , Driver type ODBC 

### Steps to reproduce

Create an SQL Server Connection, Select Driver type = ODBC , Connection type = Manual , Driver dropdown = SQL Server, Host = MyPc\SQLEXPRESS, Port = 1433, database = remove master, authentication = Windows authentication

**Comment by ShadelessFox** [comment, marked as answer]
Hello @Kevinpandya953,

May I ask why you are using ODBC driver instead of JDBC driver? The ODBC driver is not maintained and is only left for backward compatibility, whereas the JDBC driver gets regular updates from Microsoft.

**Comment by Kevinpandya953** [comment]
Hi @ShadelessFox 

Thank you for your suggestion, earlier I was trying to make a connection using ODBC which was having issues and on the other hand JDBC connection was also having some trouble connecting to SQL Server because my TCP IP protocols were not enabled in SQLEXPRESS but now it works fine. 

Thanks again for the insightful suggestion and prompt reply.
